var/spack/repos/builtin/packages/creduce/package.py

```diff
@@ -22,6 +22,7 @@ class Creduce(CMakePackage):
 
     depends_on("flex")
     depends_on("libxml2")
+    depends_on("llvm")
```

Comment:
There is no patch and `llvm` is already in the directive below (but I think the `when` condition is wrong)

Reply:
Oh, I see the description of the PR is completely outdated :laughing:

Reply:
yeah sorry, missed to update that!

thanks, @alalazo ! :)